app/templates/response/add_instruction.js.html

```diff
@@ -0,0 +1,98 @@
+<script type="text/javascript">
+$(document).ready(function () {
+    // Hides all other divs except for the first
+    $(".instruction-control .instruction-divs").each(function (e) {
+        if (e != 0)
+            $(this).hide();
+    });
+
+    // Handles click events on the first next button
+    $("#instruction-next-1").click(function () {
+        // Onclick, changes button text from to Next
+        $(this).text("Next");
+
+        // Validate instruction form
+        $("#add-instruction-form").parsley().validate();
+
+        // If validation fields are valid, proceed with next click function
+        if ($('#add-instruction-form').parsley().isValid()) {
+            var formData = {
+                request_id: "{{ request.id }}",
+                template_name: "email_response_instruction.html",
+                type: "instruction_email"
+            };
+
+            $.ajax({
+                url: "/response/email",
+                type: 'POST',
+                data: JSON.stringify(formData),
+                contentType: false,
+                success: function (data) {
+                    // Data should be html template page.
+                    tinyMCE.get('email-instruction-content').setContent(data);
+                }
+            });
+            document.getElementById("instruction-first").style.display = "none";
```

Comment:
Why do you do this instead of `$("#instruction-first").css("display", "none");` or `$("#instruction-first").hide();  // show()` in spite of the fact we are using jquery? Is there a noticeable performance gain?